contracts/vaults/RibbonThetaVault.sol

```diff
@@ -531,6 +545,20 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
         );
     }
 
+    /**
+     * @notice Optionality to set strike price manually
+     * @param strikePrice is the strike price of the new oTokens
+     */
+    function setStrikePrice(uint256 strikePrice)
+        external
+        onlyManager
+        nonReentrant
+    {
+        require(strikePrice > 0, "!strikePrice");
+        overridenStrikePrice = strikePrice;
+        lastStrikeOverride = round;
+    }
+
```

Comment:
1. updates overridenStrikePrice with new strike price
2. updates latest round which we overrode strike price. in commitAndClose if that latest round is this round then we use manual instead of strike selection contract methodology

this needs to be called before commitAndClose. so off-chain check of whether strike selection contract is acting up should be beforehand